src/wwwroot/styles/views/_settings.scss

```diff
@@ -588,7 +588,10 @@
   .my-plans--container{
     .my-plan--info-container{
       .item {
-        div {margin-top: 1em;}
+        div {          
+          margin-top: 1em;
+          .hide{display: none;}
```

Comment:
Not in all hide cases we want to put a display none, this it's only for mobile versions(the width of the device do not allow us to use a nice transition and so then we can hide it without transition).

We should have a hide-transitioned class maybe with opacity or hide by max-height, but there is so much options to hide nicely an element.